environments/icds-staging/public.yml

```diff
@@ -1,61 +1,32 @@
-couchdb2:
```

Comment:
mostly moving settings around to make it easier to compare with `icds-cas` settings